src/nsls2api/tests/api/test_beamline_api.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from httpx import AsyncClient, ASGITransport
+
+from nsls2api.main import app
+from nsls2api.models.beamlines import ServiceAccounts, Beamline
+
+
+@pytest.mark.anyio
+async def test_get_beamline_service_accounts():
+    async with AsyncClient(
+            transport=ASGITransport(app=app), base_url="http://test"
+    ) as ac:
```

Comment:
Frequent repetition of building `ac` suggests that it should become a pytest fixture.